### ISSUE TITLE
Use of unnamed union

### ISSUE THREAD

**Issue by jibsen** [centaurean/density#63]
There was [an issue](https://github.com/quixdb/squash/issues/199#issuecomment-221173870) compiling the squash density plugin, which I believe might be due to the use of an unnamed union in `src/main_header.h`. Support for this was not added until C11, but many versions of GCC supports it as an extension. If you compile density with `-pedantic`, you get the following warning:

```
../src/main_header.h:55:6: warning: ISO C99 doesn't support unnamed structs/unions [-Wpedantic]
```

**Comment by nemequ** [comment]
Note that this causes compilation to fail with GCC 4.4 (on CentOS 6, gcc 4.4.7) when compiling with `-std=c99`.  Compiling with `-std=gnu99` works, but unfortunately setting the C standard in CMake to C99 will use c99 not gnu99.

**Comment by gpnuma** [comment]
Should be fixed by 0.14.0
